CHANGELOG.md

```diff
@@ -1,28 +1,36 @@
+* Add new ``tcp_request_parser`` and ``tcp_message_delimiter`` config option. Valid values for ``tcp_request_parser``
+include ``default`` and ``batch``. New TCP recv batch oriented request parser is much more efficient than the default
+one and should be a preferred choice in most situations.  For backward compatibility reasons, the default parser hasn't
+been changed yet.
 
 Misc:
+* On startup and when parsing a config file, agent now emits a warning if the config file is readable by others.
+* Add the config option ``enable_worker_process_metrics_gather`` to enable 'linux_process_metrics' monitor for each
+multiprocess worker.
+* Each session, which runs in a separate process, periodically writes its stats in the log file.
+The interval between writes can be changed by using the ``default_worker_session_status_message_interval``
+* Rename some of the configuration parameters: ``use_miltiprocess_copying_workers``  to ``use_multiprocess_workers``,
+``default_workers_per_api_key`` to ``default_sessions_per_api_key``. Previous option names are preserved for the
+backward compatibility but they are marked as deprecated.
+NOTE: The appropriate [environment variable names](https://app.scalyr.com/help/scalyr-agent-env-aware) are changed too.
 * Update docker monitor so we don't log some non-fatal errors under warning log level when consuming logs using Docker API.
 * Add support for ``compression_type: none`` config option which completely disables compression for outgoing requests. Right now one of the main bottle necks in the high volume scenarios in the agent is compression operation. Disabling it can, in some scenarios, lead to large increase to the overall throughput (up to 2x). Disabling the compression will in most cases result in larger data egress traffic which may incur additional charges on your infrastructure provider so this option should never be set to ``none`` unless explicitly advised by the technical support.
 * Linux system metrics monitor has been updated to also ignore ``/var/lib/docker/*`` and ``/snap/*`` mount points by default. Capturing metrics for those mount points usually offers no additional insight to the end user. For information on how to change the ignore list via configuration option, please see [RELEASE_NOTES](https://github.com/scalyr/scalyr-agent-2/blob/master/RELEASE_NOTES.md).
-
-## 2.1.16 "Lasso" - December 23, 2020
-
-<!---
-Packaged by Tomaz Muraus <tomaz@scalyr.com> on Dec 23, 2020 14:00 -0800
---->
-
-Misc:
-* On startup and when parsing a config file, agent now emits a warning if the config file is readable by others.
+* The agent install bash script now adds the Scalyr repositories directly without installing the ``scalyr-repo`` packages.
```

Comment:
I would also say this resolves problems with installing on some distributions (if you have the particular distribution from the ticket, I would add it here as well.)